java/src/main/java/org/rocksdb/Transaction.java

```diff
@@ -834,11 +909,23 @@ public void delete(final byte[] key) throws RocksDBException {
    * @throws RocksDBException when one of the TransactionalDB conditions
    *     described above occurs, or in the case of an unexpected error
    */
+  public void delete(final ColumnFamilyHandle columnFamilyHandle, final byte[][] keyParts,
```

Comment:
1. Could you please rename `assume_tracked` to `assumeTracked` so it follows the Java naming conventions.

2. Could you please add the javadoc for the additional parameter `assumeTracked`.

3. This occurs in several places, could you correct each please?